tools/filter_parameters.py

```diff
@@ -0,0 +1,21 @@
+def ch_length_bounds(seq, length_bounds):
+    """Ф-ия проверки соответсвия интервалу длины рида"""
+    if type(length_bounds) is int:
+        length_bounds = [0, length_bounds]
+    return length_bounds[0] <= len(seq[0]) and len(seq[0]) <= length_bounds[1]
+
+
+def ch_gc_bounds(seq, gc_bounds):
+    """Ф-ия проверки соответсвия интервалу состава GC в процентах"""
+    if type(gc_bounds) is int:
+        gc_bounds = [0, gc_bounds]
+    GC_cont = (seq[0].count('G')+seq[0].count('C'))*100/len(seq[0])
```

Comment:
```suggestion
    GC_cont = (seq[0].count('G') + seq[0].count('C')) * 100 / len(seq[0])
```
операнды принято разделять пробелами
-1